pipeline/ingest/session_loaders.py

```diff
@@ -63,8 +67,43 @@ def load_sessions(self, subject_name):
     def load_behavior(self):
         pass
 
-    def load_tracking(self):
-        pass
+    def load_tracking(self, session_dir, subject_name, session_datetime):
+
+        # ---- identify the .mat file for tracking data ----
+        tracking_dir = session_dir / 'WhiskerTracking'
+        if not tracking_dir.exists():
+            raise FileNotFoundError(f'{tracking_dir} not found!')
+        datetime_str = datetime.strftime(session_datetime, '%Y%m%d%H%M%S')
+
+        tracking_fp = list(tracking_dir.glob(f'{subject_name}*{datetime_str}*.mat'))
+
+        if len(tracking_fp) != 1:
+            raise FileNotFoundError(f'Unable to find tracking .mat - Found: {tracking_fp}')
+        else:
+            tracking_fp = tracking_fp[0]
+
+        # ---- load .mat and extract whisker data ----
+        trk_mat = spio.loadmat(tracking_fp, struct_as_record=False, squeeze_me=True)[tracking_fp.stem]
+
+        frames = np.arange(max(trk_mat.fid) + 1)
+
+        whisker_inds = np.unique(trk_mat.wid)
+        whiskers = {wid: {} for wid in whisker_inds}
+        for wid in whisker_inds:
+            matched_wid = trk_mat.wid == wid
+            matched_fid = trk_mat.fid[matched_wid]
+            _, matched_frame_idx, _ = np.intersect1d(matched_fid, frames, return_indices=True)
+
+            for var in ('follicle_x', 'follicle_y', 'angle'):
+                d = np.full(len(frames), np.nan)
+                d[matched_frame_idx] = getattr(trk_mat, var)[matched_wid]
+                whiskers[wid][var] = d
+
```

Comment:
not sure of the input data, could make sense to throw some asserts in here to sanity check shapes if applicable